### ISSUE TITLE
[Package Request] - pandoc

### ISSUE THREAD

**Issue by neilcook** [amazonlinux/amazon-linux-2022#379]
**What package is missing from Amazon Linux 2023? Please describe and include package name.**

pandoc

Pandoc is a package that converts between many different markup formats.

**Is this an update to existing package or new package request?**

New package request

**Is this package available in Amazon Linux 2? If it is available via external sources such as EPEL, please specify.**

It is available in Amazon Linux 2 via EPEL

**Any additional information you'd like to include. (use-cases, etc)**

For producing documentation, pandoc is invaluable.

**Comment by stewartsmith** [comment]
I agree that `pandoc` can be useful. IIRC it did end up having a *very* large dependency graph that had to be brought in, so it may be trickier than some other packages...

**Comment by stewartsmith** [comment]
At this point in time, we're not looking at adding Haskell (and thus `pandoc`) to AL2023.

Note that if anyone in the community wants to build and maintain packages in their own for AL2023, [Fedora COPR](https://copr.fedorainfracloud.org/) does support building AL2023 packages and repos, so using that community infrastructure is a possibility.

**Comment by daniejstriata** [comment]
I was able to build pandoc in COPR when using the Fedora 36 repo and building binutils-2.40-14.fc39.src.rpm beforehand as the binutils included with AL2023 does not include ld gold linker. 
https://copr.fedorainfracloud.org/coprs/faramirza/al2023_build-dependencies/build/7768736/

**Comment by ozbenh** [comment]
Why would it require gold ? As far as we know, gold is more or less deprecated and unmaintained, we took it out on purpose

**Comment by daniejstriata** [comment]
I'm unsure how to change the pandoc src rpm so not need the gold linker. Seems that Fedora is using the gold linker for all their GHC builds as none of them I looked at wanted to build without the gold linker.

**Comment by Vlaaaaaaad** [comment]
FYI: apparently, pandoc is now available as part of the newly-released [_"Supplementary Packages for Amazon Linux (SPAL)_"](https://docs.aws.amazon.com/linux/al2023/ug/spal.html) available from Amazon Linux 2023 (AL2023) version 2023.9.20251117 onward:
> Supplementary Packages for Amazon Linux (SPAL) is a **dedicated package repository that provides access to thousands of additional packages derived from Extra Packages for Enterprise Linux 9 (EPEL9)**. These packages complement the existing software available in core Amazon Linux 2023.
>
> SPAL simplifies software deployment by providing pre-built packages that are compatible with AL2023, eliminating the need for customers to build packages from source code themselves. This saves time and effort in the software installation process.

[...]

> SPAL provides several key advantages to Amazon Linux 2023 users:
> - Expanding AL2023 use cases — Access to popular additional packages beyond the core AL2023 repository, such as **pandoc**, GDAL or drbd-utils, enables customers to support multiple business and development needs.
> - Simplifying AL2023 package management — By providing packages pre-built for AL2023, the process of building additional packages from source is eliminated, saving time and reducing the risk of compilation errors.
> - Streamlining AL2 to AL2023 migration — SPAL repository enables seamless migration of workloads from AL2 to AL2023, including workloads that depend on EPEL7 packages.

[...]

>SPAL packages are **not covered by the same level of support as core AL2023 packages**, which receive support for the entire life of Amazon Linux 2023.
>
> **Important**: Prior to using SPAL, customers must carefully evaluate the following considerations:
>   - SPAL packages are NOT covered by AWS Enterprise Support.
>   - SPAL packages are provided 'as-is' from upstream EPEL9.
>   - SPAL packages will NOT receive AWS CVE security tracking.
>   - SPAL packages receive security patches and bug fixes exclusively from upstream EPEL9 when available.

**Comment by SreeSunku** [comment]
Thank you for your patience. This package is now available as part of the SPAL Repository for AL2023. Please use these resources to familiarize yourself on installing SPAL repository and the packages.  [SPAL Technical Documentation](https://docs.aws.amazon.com/linux/al2023/ug/configure-spal-repository.html), [SPAL list of packages](https://docs.aws.amazon.com/linux/al2023/release-notes/al-2023-spal-packages.html).
